public/components/schemas/ConnectionUpdate.yml

```diff
@@ -0,0 +1,49 @@
+type: object
+properties:
+  from:
+    description: Source entity, must be used with to and cannot be used with entity1 or entity2 
```

Comment:
From your descriptions it is impossible to understand the difference between from and entity1 for example. Why should I use one vs the other? Why do we even have two fields?